changelogs/changelog.yaml

```diff
@@ -1,5 +1,11 @@
 ancestor: null
 releases:
+  1.1.0:
+    changes:
+      minor_changes:
+        - Adds an ability to specify which ProxySQL check type to use when detecting
+          a MySQL standby node in proxysql_replication_hostgroups module.
```

Comment:
Please do NOT modify `changelogs/changelog.yaml` directly. Create a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).